### PR TITLE
Clean up UI tests

### PR DESCRIPTION
**Pull request by davidbrochart**

### CODE REVIEW

File: .github/workflows/ui-tests.yml [hunk 1]
```diff
@@ -29,18 +29,17 @@ jobs:
           cd ui-tests
           jlpm playwright install chromium
 
-      - name: Launch JupyterLab 
+      - name: Launch JupyterLab
         run: |
           cd ui-tests
-          jlpm start-jlab:detached 2>&1 > /tmp/jupyterlab_server.log &          
+          jlpm start-jlab:detached > /tmp/jupyterlab_server.log 2>&1
```
Comment (davidbrochart) [inline .github/workflows/ui-tests.yml:35]: It was not redirecting standard output and error.

File: .github/workflows/ui-tests.yml [hunk 2]
```diff
@@ -29,18 +29,17 @@ jobs:
...
 
       - name: Wait for JupyterLab
         uses: ifaxity/wait-on-action@v1
         with:
           resource: http-get://localhost:8888/
           timeout: 360000
 
-      - uses: iterative/setup-cml@v1
```
Comment (davidbrochart) [inline .github/workflows/ui-tests.yml:43]: I don't think we need this.

**Comment by davidbrochart** [comment]
@trungleduc do you agree with these changes?

**Comment by trungleduc** [comment]
Yes, thanks for the updates!

**Pull request merged**